e2e/test_android/login/login_spec.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// *********************************************************************
+// - [#] indicates a test step (e.g. # Go to a screen)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use accessibility ID when selecting an element. Create one if none.
+// *********************************************************************
+
+import SelectServerScreen from '../../screen_objects/select_server_screen';
+import LoginScreen from '../../screen_objects/login_screen';
+import ChannelScreen from '../../screen_objects/channel_screen';
```

Comment:
is there a way to make this imports not be relative ?

Reply:
I'm afraid no 'cause we're not and probably will not be doing code transformation for those E2E scripts.  I looked at other repos and I don't see they are doing it either.  But I could be wrong and making it relative would be really nice. By the way, we have the same case on Cypress testing in the webapp.